### DISCUSSION TITLE
Linha de comando

### DISCUSSION THREAD

**Discussion by Matheusmauriciom** [balta-io/forum#489] [Tira Dúvidas] [answered]
![Screenshot_3](https://github.com/balta-io/forum/assets/150610041/247b96f7-75dd-48a5-9c0d-ece16958c940)

Qual a linha de comando para criar o novo template do blazor? (server e was)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Matheusmauriciom, como vai?
```
dotnet new blazor -o MeuApp -int Auto
```
